axiom-cloudwatch-lambda/handler.py

```diff
@@ -0,0 +1,136 @@
+import base64
+import gzip
+import json
+import logging
+import os
+import re
+import urllib
```

Comment:
not sure why it worked on https://github.com/axiomhq/axiom-cloudfront-lambda. but it keeps throwing me error like this:
```
AttributeError: module 'urllib' has no attribute 'request'
```
you have to import urllib.request